docs/getting-started/quickstart.md

```diff
@@ -26,6 +26,8 @@ It will download everything that is needed to start n8n.
 You can then access n8n by opening:
 [http://localhost:5678](http://localhost:5678)
 
+**Note:** The minimum Node.js version required is v14. Make sure to update Node.js to v14 or above.
```

Comment:
@harshil1712 Could you please check if this is correct? This page lists the minimum version as 14.15: https://docs.n8n.io/nodes/creating-nodes/create-node.html#prerequisites